test/CoveragePool.test.js

```diff
@@ -213,6 +293,20 @@ describe("CoveragePool", () => {
             )
           ).to.be.above(0)
         })
+
+        it("should emit RiskManagerApprovalStarted event", async () => {
```

Comment:
s/RiskManagerApprovalStarted/RiskManagerUnapprovalStarted

Reply:
Fixed in d0c89b8f2b88677758133e85a80e456e32fdbb0b.